pip_run/commands.py

```diff
@@ -2,7 +2,7 @@
 import contextlib
 import pathlib
 
-from jaraco import env  # type: ignore # (python/mypy#15970)
+import jaraco.env as env
```

Comment:
Let's keep the more elegant syntax. Unless the Python language itself seeks to deprecate that more elegant syntax, the false positive is a bug in mypy. This project shouldn't bear that burden. Done in dee42fc.